src/components/ColorPicker/components/ColorPickerItemComponent/ColorPickerItemComponent.jsx

```diff
@@ -55,35 +75,56 @@ const ColorPickerItemComponent = ({
   const shouldRenderIcon = shouldRenderSelectedIcon || ColorIndicatorIcon;
   const colorIndicatorWrapperStyle = shouldRenderIndicatorWithoutBackground ? { color: colorAsStyle } : {};
+
+  const backgroundStyle = useMemo(() => {
+    if (isColorPickerItem) return null;
+    return shouldRenderIndicatorWithoutBackground ? "transparent" : colorAsStyle;
+  }, [colorAsStyle, isColorPickerItem, shouldRenderIndicatorWithoutBackground]);
+
   return (
-    <Tooltip content={tooltipContent}>
-      <li
-        className={cx("monday-style-color-item-wrapper", {
-          "selected-color": isSelected,
-          active: isActive,
-          circle: colorShape === COLOR_SHAPES.CIRCLE
-        })}
-      >
-        <div className="feedback-indicator" />
-        <Clickable
-          ref={itemRef}
-          ariaLabel={color}
-          className={cx("color-item", `color-item-size-${colorSize}`, {
-            "color-item-text-mode": shouldRenderIndicatorWithoutBackground
+    <div className="monday-style-color-item-container">
+      <Tooltip content={tooltipContent}>
+        <li
+          className={cx("monday-style-color-item-wrapper", {
+            "selected-color": isSelected,
+            active: isActive,
+            circle: colorShape === COLOR_SHAPES.CIRCLE
           })}
-          style={{ background: shouldRenderIndicatorWithoutBackground ? "transparent" : colorAsStyle }}
-          onClick={onClick}
-          tabIndex="-1"
-          onMouseDown={onMouseDown} // this is for quill to not lose the selection
         >
-          <div className="color-indicator-wrapper" style={colorIndicatorWrapperStyle}>
-            {shouldRenderIcon && (
-              <Icon icon={shouldRenderSelectedIcon ? SelectedIndicatorIcon : ColorIndicatorIcon} ignoreFocusStyle />
-            )}
-          </div>
-        </Clickable>
-      </li>
-    </Tooltip>
+          <div className="feedback-indicator" />
+          <Clickable
+            ref={itemRef}
+            ariaLabel={color}
+            className={cx("color-item", `color-item-size-${colorSize}`, {
+              "color-item-text-mode": shouldRenderIndicatorWithoutBackground,
+              rainbow: isColorPickerItem
+            })}
+            style={{
+              background: backgroundStyle
+            }}
+            onClick={isColorPickerItem ? toggleColorPicker : onClick}
+            tabIndex="-1"
+            onMouseDown={onMouseDown} // this is for quill to not lose the selection
+          >
+            <div className="color-indicator-wrapper" style={colorIndicatorWrapperStyle}>
+              {shouldRenderIcon && (
+                <Icon icon={shouldRenderSelectedIcon ? SelectedIndicatorIcon : ColorIndicatorIcon} ignoreFocusStyle />
+              )}
+            </div>
+          </Clickable>
+        </li>
+      </Tooltip>
+      {isColorPickerItem && isSelected && (
+        <div className="clear-color-picker" onClick={() => clearSelectedColor(customColor)}>
```

Comment:
how will keyboard users be able to interact with this button?
(BTW, consider using a button and not a div, so it will be tab-able..)